<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RouterManagerWindow</class>
 <widget class="QMainWindow" name="RouterManagerWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>520</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Router Management</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QTabWidget" name="tab">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_hosts">
       <attribute name="title">
        <string>Hosts</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_title_hosts_conn_count">
            <property name="text">
             <string>Connections count:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_hosts_conn_count">
            <property name="text">
             <string notr="true">0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QSplitter" name="splitter">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <widget class="QTreeWidget" name="tree_hosts">
           <property name="indentation">
            <number>0</number>
           </property>
           <column>
            <property name="text">
             <string>Computer Name</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>IP Address</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Connect Time</string>
            </property>
           </column>
          </widget>
          <widget class="QTreeWidget" name="tree_host_info">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <column>
            <property name="text">
             <string>Name</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Value</string>
            </property>
           </column>
          </widget>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="button_disconnect_host">
            <property name="text">
             <string>Disconnect</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_disconnect_all_hosts">
            <property name="text">
             <string>Disconnect All</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_refresh_hosts">
            <property name="text">
             <string>Refresh</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_relay">
       <attribute name="title">
        <string>Relay</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_title_relay_conn_count">
            <property name="text">
             <string>Connections count:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_relay_conn_count">
            <property name="text">
             <string notr="true">0</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTreeWidget" name="tree_relay">
          <property name="indentation">
           <number>0</number>
          </property>
          <column>
           <property name="text">
            <string>Address</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Connect Time</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Pool Size</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Version</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Computer Name</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Operating System</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="button_refresh_relay">
            <property name="text">
             <string>Refresh</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_users">
       <attribute name="title">
        <string>Users</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="button_add_user">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>:/img/plus.png</normaloff>:/img/plus.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_modify_user">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>:/img/pencil.png</normaloff>:/img/pencil.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_delete_user">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>:/img/minus.png</normaloff>:/img/minus.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="button_refresh_users">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../resources/client.qrc">
              <normaloff>:/img/arrow-circle-double.png</normaloff>:/img/arrow-circle-double.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>198</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTreeWidget" name="tree_users">
          <property name="indentation">
           <number>0</number>
          </property>
          <column>
           <property name="text">
            <string>User Name</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="../resources/client.qrc"/>
 </resources>
 <connections/>
</ui>
